src/style.css

```diff
@@ -1 +1,40 @@
 /* add styles here */
+body {
+margin: 0;
+}
+
+.stars {
+display: flex;
+justify-content: center;
+align-items: center;
+}
+
+.stars__star {
+margin-right: 4px;
+width: 16px;
+height: 16px;
+background-image: url(./images/star.svg);
+background-repeat: no-repeat;
+background-position: center;
+flex-shrink: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+background-image: url(./images/star-active.svg);
+}
```

Comment:
combine selectors using comma,  example below:

.selector1,
.selector2 {
  your styles here...
}

Reply:
When I remove margin-right from the last stars in a row, i use the following: 
.stars__star:last-child {
  margin: 0;
} ...and after that I can't pass any tests.